src/components/form-group/package.json

```diff
@@ -17,12 +17,12 @@
     "@ecl/range": "4.9.0",
     "@ecl/rating-field": "4.9.0",
     "@ecl/select": "4.9.0",
-    "@ecl/twig-component-text-area": "4.9.0",
-    "@ecl/twig-component-text-input": "4.9.0"
+    "@ecl/text-area": "4.9.0",
+    "@ecl/text-input": "4.9.0"
   },
   "devDependencies": {
-    "@ecl/specs-component-text-area": "4.9.0",
-    "@ecl/specs-component-text-input": "4.9.0"
+    "@ecl/text-area": "4.9.0",
```

Comment:
it seems these are already dependencies, so we don't need to duplicate them as devDependencies